hazelcast-jclouds/src/test/java/com/hazelcast/jclouds/integration/AWSEC2LiveTest.java

```diff
@@ -0,0 +1,50 @@
+package com.hazelcast.jclouds.integration;
+
+import com.hazelcast.test.HazelcastSerialClassRunner;
+import com.hazelcast.test.annotation.LiveTest;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Category(LiveTest.class)
+@RunWith(HazelcastSerialClassRunner.class)
+public class AWSEC2LiveTest extends AbstractLiveTest {
+
+    public static LiveComputeServiceBuilder builder =
+            new LiveComputeServiceBuilder(AbstractLiveTest.AWS_PROVIDER);
+
+
+    @BeforeClass
+    public static void provisionNodes() throws Exception {
+        builder.provisionNodes();
+    }
+
+    @AfterClass
+    public static void destroyNodes() throws Exception {
+        builder.destroyNodes();
```

Comment:
When destroy operation nodes fails, we might have leftover nodes. Right now, I could not integrate Harakiri monitor because it seem to be a separate jvm process started by this script, https://github.com/hazelcast/hazelcast-simulator/blob/master/dist/src/main/dist/bin/harakiri-monitor
but I will find a way to start  same kind of monitoring mechanism for our tests too. but not in this PR.